packages/block-library/src/breadcrumbs/index.php

```diff
@@ -25,27 +25,26 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 	$breadcrumb_items = array();
 
 	if ( $attributes['showHomeLink'] ) {
-		$breadcrumb_items[] = sprintf(
-			'<a href="%s">%s</a>',
-			esc_url( home_url() ),
+		$breadcrumb_items[] = block_core_breadcrumbs_create_link(
+			home_url(),
```

Comment:
While we're here, let's avoid the extra redirect to a URL with forward slash by adding: 

```suggestion
			home_url( '/' ),
```

Reply:
I had this change here: https://github.com/WordPress/gutenberg/pull/72839, but I guess it's fine to add it here too. There will be conflicts to rebase either way :)

Reply:
Ah, right. Fine to do it in any of the PRs 😉